test/e2e_kubeadm/networking_test.go

```diff
@@ -122,6 +121,39 @@ var _ = KubeadmDescribe("networking [setup-networking]", func() {
 			})
 		})
 	})
+	ginkgo.Context("dual-stack [Feature:IPv6DualStack]", func() {
+		ginkgo.Context("podSubnet", func() {
+			ginkgo.It("should be properly configured if specified in kubeadm-config", func() {
+				if !dualStack {
+					framework.Skipf("Skipping because cluster is not dual-stack")
+				}
+				if !podSubnetInKubeadmConfig {
+					framework.Skipf("Skipping because podSubnet was not specified in kubeadm-config")
+				}
+				cc := getClusterConfiguration(f.ClientSet)
+				if _, ok := cc["networking"]; ok {
+					netCC := cc["networking"].(map[interface{}]interface{})
+					if ps, ok := netCC["podSubnet"]; ok {
+						nodes, err := f.ClientSet.CoreV1().Nodes().List(metav1.ListOptions{})
+						framework.ExpectNoError(err)
+						// Check that the pod CIDRs allocated to the node(s) are within the kubeadm-config podCIDR.
+						configCIDRs := strings.Split(ps.(string), ",")
+						for _, cCIDR := range configCIDRs {
+						NODE_LOOP:
```

Comment:
Is there a way to avoid this label inside a loop?
What if we loop on nodes, and inside this we loop on config CIDRs?

Reply:
Done!